dom.bs

```diff
@@ -1541,6 +1541,10 @@ itself.</p>
 <p>An <a for=/>element</a> is <dfn export>in a shadow-including document</dfn> if its
 <a>shadow-including root</a> is a <a>document</a>.
 
+<p>An object <var>A</var> and an object <var>B</var> are <dfn export>in the same shadow-including
+document</dfn> if both <var>A</var> and <var>B</var> are <a>in a shadow-including document</a> and
+<var>A</var> and <var>B</var> share the same <a>shadow-including root</a>.
```

Comment:
Apparently "in a shadow-including document" is only for elements. Is that wrong or should this also be for elements only? Maybe "node" for both makes more sense?